gramps/gui/editors/editplace.py

```diff
@@ -195,6 +195,11 @@ def get_latlongitude(self):
         return ""
 
     def _validate_coordinate(self, widget, text, typedeg):
+        if text.find(',') != -1:
+            # coordinates must be in english format. 43,2545 is not allowed.
+            return ValidationError(
+                _('Invalid latitude\n(syntax: '
```

Comment:
But it should say `longitude` for longitudes? Maybe add this condition to the below checks instead.

Reply:
conv_lat_lon call __convert_float_val which convert the comma. I think this is the solution.
needs more tests because conv_lat_lon is used in many places. If we have already commas in lat/lon, we could have problems.